licenses/_headers/license-prepend-missing.sh

```diff
@@ -16,10 +16,13 @@
 # Get the source dir and ignore variables
 source license-setup.sh
 
+GET=( -iname "*.py" -or -iname "*.sh" -or -iname "*.bash" -or -iname "*.csh" \
+     -or -iname "*.pbs" -or -iname "*.html" -or -iname "*.css" -or -iname "*.js" )
```

Comment:
@jhkennedy, this was added because we have files in our repo that don't match any of these and they were getting listed as files where a license would be added but that didn't actually happen.

Reply:
Ah, right, b/c the find all files is more inclusive than the sum of the prepend commands. This is a good addition, but it mean `license-find-missing.sh` will also return a more inclusive list than using this script. So, it'd probably be a good idea to either: 

1. include these switches in `license-find-missing.sh` -- inserting `\( "${GET[@]}" \) ` into the find command, and including `GET=` at the top or in `license-setup.sh`

2. Keep the more inclusive find command but also print a list of files that have no rule defined to prepend the license header -- At the top of `license-prepend-missing.sh` say `ATTEMPTING TO PREPEND A LICENSE...` and then at the bottom list the files that no action was performed [possibly by just running `license-find-missing.sh` again], with a warning message to either add those files to the ignore lists or define a rule for them.